src/views/Home.vue

```diff
@@ -207,6 +208,20 @@ export default defineComponent({
     const canUndo = computed(() => store.getters[CAN_UNDO]);
     const canRedo = computed(() => store.getters[CAN_REDO]);
 
+    // add hotkeys
+    Mousetrap.bind(["ctrl+s"], () => {
+      generateAndSaveAllAudio();
+    });
+
+    Mousetrap.bind(
+      ["+"],
```

Comment:
With a Japanese keyboard, the user has to press two keys at the same time to get the "+"....
How about the Enter key?